.github/workflows/test_and_deploy.yml

```
@@ -14,62 +14,78 @@ jobs:
         with:
           node-version: lts/iron
 
+        # We cannot use build in LFS due to missing support of custom URLs: https://github.com/actions/checkout/issues/365
       - name: Pull Repository with LFS
         run: git lfs pull
 
       - name: Install
-        uses: bahmutov/npm-install@v1
+        run: npm ci
 
       - name: Build and package library
         run: |
```

why use the `|` when only one, short command is there.

This is the same for all occurrences below.

```suggestion
        run: npm run package
```